src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig

```diff
@@ -53,9 +53,26 @@
 
 {% block form_row -%}
   {% spaceless %}
+    {% import '@PrestaShop/Admin/macros.html.twig' as ps %}
+    {% set multistoreCheckboxName = 'multistore_' ~ form.vars.name %}
+    {% set disabledField = false %}
+    {% set labelClass = '' %}
+    {% if form.vars.attr.disabled is defined and form.vars.attr.disabled %}
+      {% set disabledField = true %}
+      {% set labelClass = 'disabled' %}
+    {% endif %}
+
     <div class="{{ block('form_row_class') }}{% if (not compound or force_error|default(false)) and not valid %} has-error{% endif %}"{% if (attr.visible is defined and not attr.visible) %} style="display: none;"{% endif %}>
-      {{ form_label(form) }}
-      <div class="{{ block('form_group_class') }}">
+      {% if attribute(form.parent, multistoreCheckboxName) is defined %}
+        {{ form_errors(attribute(form.parent, multistoreCheckboxName)) }}
+        {{ form_widget(attribute(form.parent, multistoreCheckboxName)) }}
+      {% endif %}
+      {% if form.vars.help is defined %}
+        {{ ps.label_with_help(form.vars.label, form.vars.help, labelClass) }}
+      {% else %}
```

Comment:
The blue question box is supposed to go away and we should only use help at the bottom of the input, so importing macros here is not needed. Instead, you can use  {{ block('form_help') }} to display help.

Also I try to include help as part of every widget, so make sure help here doesn't get duplicated.